assets/third_party/inputs/atlas_kenney_input_prompts_1.4/import_spritesheets.gd

```diff
@@ -17,7 +17,7 @@ const SHEETS := [
 ]
 
 # Whether to import new assets, or just refresh existing ones
-const IMPORT_NEW := false
+const IMPORT_NEW := true
```

Comment:
I think that you commited this flag change by accident. In a previous commit you say:
> added a switch (disabled by default) that controls whether new textures are created, or only existing
ones updated.

 Not a big deal though, this is internal.